src/main/index.js

```diff
@@ -816,6 +822,21 @@ function runApp() {
             accelerator: 'CmdOrCtrl+Shift+R'
           },
           { role: 'toggledevtools' },
+          { role: 'toggledevtools', accelerator: 'f12', visible: false },
+          {
+            label: 'Enter Inspect Element Mode',
+            accelerator: 'CmdOrCtrl+Shift+C',
+            click: (_, window) => {
+              if (window.webContents.isDevToolsOpened()) {
+                window.devToolsWebContents.executeJavaScript('DevToolsAPI.enterInspectElementMode()')
+              } else {
+                window.webContents.once('devtools-opened', () => {
+                  window.devToolsWebContents.executeJavaScript('DevToolsAPI.enterInspectElementMode()')
+                })
+                window.webContents.openDevTools()
+              }
+            }
```

Comment:
I don't know if we really need this keyboard shortcut in the FreeTube windows, as you can use this keyboard shortcut in the devtools windows themselves already. I just added this as it was in electron-debug and I wasn't sure if anyone was actively using it (I wasn't).

Reply:
I am not

Reply:
Also not me